import_export/resources.py

```diff
@@ -415,6 +415,12 @@ def after_import_instance(self, instance, new, **kwargs):
         """
         pass
 
+    def get_connection_name(self):
```

Comment:
get_connection_name defined twice

Reply:
If you're looking at the diff, it's not defined twice in the same location - it's defined twice in multiple classes. I honestly don't remember why I did it this way - it's been a while...

Reply:
ha yes i realized as i wrote this that this is a couple hundred commits behind master! i ended up forking it myself and doing a similar thing to your fork a couple years ago, really needed multi-DB!